acc_test/resource_cluster_test.go

```diff
@@ -40,6 +42,34 @@ func AccClusterResourceTestSteps(t *testing.T, testAccProviders map[string]terra
 					testCheckResourceCreated("valid", hostname, username, apiKey),
 				),
 			},
+			{
+				Config: oldToNewVersionConfig,
+				Check: resource.ComposeTestCheckFunc(
+					testCheckResourceValid("valid"),
+				),
+				PlanOnly: true,
+			},
+			{
+				Config: oldToNewVersionConfig,
```

Comment:
Why would old to new destroy the cluster? Or am I reading this test step wrong?

Reply:
It needs to be destroyed and re-created in order to change the format of the version string to check the inverse trasition between version formats.